server/middlewares/AuthValidator.js

```diff
@@ -77,6 +77,24 @@ class AuthValidator {
     }
     return next();
   }
+
+  static isAdmin(req, res, next) {
+    try {
+      const authorization = req.headers.authorization.split(' ')[1] || req.headers.token;
+
+      if (!authorization) {
+        return res.status(401).json({ error: true, message: 'Unauthorized' });
```

Comment:
I see that you have many unauthorized messages, Kindly give a more descriptive message

Reply:
Okay. I'll do that now. Thanks.

Reply:
I've implemented the feedback above. Thank you